elixir/markdown/lib/markdown.ex

```diff
@@ -49,9 +49,9 @@ defmodule Markdown do
     "<p>#{join_words_with_tags(split_line)}</p>"
   end
 
-  # I changed the argument to unsplit_text
+  # I changed the argument to unsplit_text and used Enum.map_join instead of nested functions
   defp join_words_with_tags(unsplit_text) do
-    Enum.join(Enum.map(unsplit_text, fn word -> replace_md_with_tag(word) end), " ")
+    Enum.map_join(unsplit_text, " ", fn word -> replace_md_with_tag(word) end)
```

Comment:
Very nice, one last improvement: when there is just a single argument for the function inside of the anonymous function use a function capture instead:

```suggestion
    Enum.map_join(unsplit_text, " ", &replace_md_with_tag/1)
```

Reply:
Yes, thank you! I actually already did that. not sure, why it was still showing the old version here.